### PR TITLE
Fixes notebook

### PR DESCRIPTION
**Pull request by madil90**
Fixes notebooks and example (#97).

### Description
- Fix all imports.
- Add comments/quality impovements.
- Evaluation now logs metrics to tensorboard.
- Tune parameters for good loss/metric graphs.

### Status
**Ready**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: examples/unet_segmentation_3d.py [hunk 1]
```diff
@@ -134,30 +135,47 @@ def log_training_loss(engine):
                                      255, engine.state.epoch)
     engine.logger.info("Epoch[%s] Loss: %s", engine.state.epoch, engine.state.output[1])
 
-
-loader = DataLoader(ds, batch_size=20, num_workers=8, pin_memory=torch.cuda.is_available())
-val_loader = DataLoader(ds, batch_size=20, num_workers=8, pin_memory=torch.cuda.is_available())
 writer = SummaryWriter()
```
Comment (ericspod) [inline examples/unet_segmentation_3d.py]: This should be defined before the event function which uses it. It still works with Python's scoping behaviour but it makes more sense when defined first.

File: examples/unet_segmentation_3d.py [hunk 2]
```diff
@@ -34,16 +37,13 @@
...
-
-config.print_config()
+monai.config.print_config()
 
 # Create a temporary directory and 50 random image, mask paris
 tempdir = tempfile.mkdtemp()
 
 for i in range(50):
-    im, seg = create_test_image_3d(256, 256, 256)
+    im, seg = create_test_image_3d(128, 128, 128)
```
Comment (wyli) [inline examples/unet_segmentation_3d.py]: this function by default generates `seg` with 5 foreground classes, I think we need 
```im, seg = create_test_image_3d(128, 128, 128, num_seg_classes=1)```
so that it's consistent with the network's `num_classes=1` setting

File: examples/unet_segmentation_3d.ipynb [hunk 3]
```diff
@@ -146,90 +123,336 @@
...
+    "trainer.add_event_handler(event_name=Events.EPOCH_COMPLETED,\n",
+    "                          handler=checkpoint_handler,\n",
+    "                          to_save={'net': net, 'opt': opt})\n",
+    "train_stats_handler = StatsHandler()\n",
+    "train_stats_handler.attach(trainer)\n",
     "\n",
     "@trainer.on(Events.EPOCH_COMPLETED)\n",
     "def log_training_loss(engine):\n",
-    "    print(\"Epoch\", engine.state.epoch, \"Loss:\", engine.state.output)\n",
+    "    # log loss to tensorboard with second item of engine.state.output, loss.item() from output_transform\n",
```
Comment (wyli) [inline examples/unet_segmentation_3d.ipynb]: perhaps put some instructions here about the training curves/prediction visualisations using `tensorboard --logdir  runs/` after running the trainer

File: examples/unet_segmentation_3d.py (hunk 1 above)
Comment (madil90) [reply examples/unet_segmentation_3d.py]: Fixed

File: examples/unet_segmentation_3d.py (hunk 2 above)
Comment (madil90) [reply examples/unet_segmentation_3d.py]: Fixed

File: examples/unet_segmentation_3d.ipynb (hunk 3 above)
Comment (madil90) [reply examples/unet_segmentation_3d.ipynb]: Added tensorboard display (inside notebook) as new section.
